src/containerapp/azext_containerapp/_clients.py

```diff
@@ -27,6 +27,7 @@
 logger = get_logger(__name__)
 
 PREVIEW_API_VERSION = "2023-08-01-preview"
+PREVIEW_API_VERSION_2023_11_02 = "2023-11-02-preview"
```

Comment:
There is a PR upgrade the api-version to 2023-11-02-preview going to merge, [build_env_vars](https://github.com/Azure/azure-cli-extensions/pull/7207/files)